src/phone_number/mobile.rs

```diff
@@ -56,7 +59,7 @@ pub static IRAN_MOBILE_OPERATORS: phf::Map<&'static str, &'static [&'static str]
 
 };
 
-#[derive(Debug, PartialEq, Eq)]
```

Comment:
@robatipoor Maybe I need to return "Eq"s back. I don't know if you have some near-future use cases in your mind or not.